nodebuilder/share/share.go

```diff
@@ -17,11 +20,30 @@ import (
 
 var _ Module = (*API)(nil)
 
-// GetRangeResult wraps the return value of the GetRange endpoint
-// because Json-RPC doesn't support more than two return values.
-type GetRangeResult struct {
-	Shares []share.Share
-	Proof  *types.ShareProof
+// RangeResult wraps the return value of the GetRange endpoint.
+// It contains a set of shares along with their proof to
+// the data root.
+type RangeResult struct {
```

Comment:
We should call it `Range`. The `Result` implies it's an API-specific struct, while it's not. Range is gonna become a shared type

Reply:
but it's currently the return value of the GetRange endpoint. You mean there is separate work that will make RangeResult used elsewher?

Reply:
Let's keep it as it is.